tt_metal/impl/program/program.hpp

```diff
@@ -103,6 +104,9 @@ class Program {
     void allocate_circular_buffers();
 
    private:
+    ProgramDeviceMap program_device_map;
+    std::unique_ptr<Buffer> buffer;
```

Comment:
could you rename this or add a comment explaining this buffer holds the program so it can be transferred to device

Reply:
Added comment